types/elsys/ems-door/default.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "https://akenza.io/elsys/ems-door/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "accX": {
```

Comment:
does this sensor really have an accelerometer?

Reply:
Yes